### ISSUE TITLE
Library to handle communicating with Rackspace API?

### ISSUE THREAD

**Issue by parkr** [mattr-/hubot-rackspace-dns#2]
Found [pkgcloud](https://github.com/nodejitsu/pkgcloud), what do you think?

**Comment by mattr-** [comment]
:sparkles: if it'll work with the DNS stuff you have, then :+1:

**Comment by parkr** [comment]
It occurs to me that I have no idea how to test either the API calls or the robot API. Perhaps @troyswanson has some insight into the former?

**Comment by troyswanson** [comment]
I wish, bro. I'm watching you guys to learn, too!

**Comment by mattr-** [comment]
So @pzula and I mucked around with both [pkgcloud](https://github.com/nodejitsu/pkgcloud) and the Cloud DNS API last night in a fun bit of 
![dinosaur-hands](https://f.cloud.github.com/assets/77174/2208751/72e8625e-9983-11e3-9b07-ba895c888388.gif)

and so while I still have no idea how to test this thing, I at least feel a bit better about what's going on. There were a couple of 
![aha](https://f.cloud.github.com/assets/77174/2208761/9ac16d34-9983-11e3-88ec-544a37155602.gif)
moments in there too. 

pkgcloud is pretty awesome and I went ahead and merged the branch that pulled in that dependency since I think that's what we're want to use.
